lib/ui/useable-components/order/index.tsx

```
@@ -1,242 +1,451 @@
+                                </Text>
+                              </View>
+
+                              {item?.addons?.length > 0 && (
+                                <View className="w-[90%]">
+                                  <Text className="font-[Inter] text-md  text-gray-600">
+                                    Addons (Options)
+                                  </Text>
+
+                                  {item?.addons?.map((addon, id) => {
+                                    return (
+                                      <View>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)